Makefile

```diff
@@ -2,39 +2,85 @@
 # Image URL to use all building/pushing image targets
 IMG ?= goharbor/harbor-operator:dev
 
-SHELL = /bin/sh
-
+CONFIGURATION_FROM ?= file:$(CURDIR)/config-dev.yml
+export CONFIGURATION_FROM
+
+REGISTRY_TEMPLATE_PATH ?= $(CURDIR)/config/manager/assets/registry-config.yaml.tmpl
+REGISTRYCTL_TEMPLATE_PATH ?= $(CURDIR)/config/manager/assets/registryctl-config.yaml.tmpl
+JOBSERVICE_TEMPLATE_PATH ?= $(CURDIR)/config/manager/assets/jobservice-config.yaml.tmpl
+CORE_TEMPLATE_PATH ?= $(CURDIR)/config/manager/assets/core-config.conf.tmpl
+CHARTMUSEUM_TEMPLATE_PATH ?= $(CURDIR)/config/manager/assets/chartmuseum-config.yaml.tmpl
+NOTARY_SERVER_TEMPLATE_PATH ?= $(CURDIR)/config/manager/assets/notary-server-config.json.tmpl
+NOTARY_SIGNER_TEMPLATE_PATH ?= $(CURDIR)/config/manager/assets/notary-signer-config.json.tmpl
+
+export REGISTRY_TEMPLATE_PATH
+export REGISTRYCTL_TEMPLATE_PATH
+export JOBSERVICE_TEMPLATE_PATH
+export CORE_TEMPLATE_PATH
+export CHARTMUSEUM_TEMPLATE_PATH
+export NOTARY_SERVER_TEMPLATE_PATH
+export NOTARY_SIGNER_TEMPLATE_PATH
+
+# See https://github.com/settings/tokens for GITHUB_TOKEN. No permissions required.
+NOTARY_SERVER_MIGRATION_SOURCE ?= github://holyhope:$${GITHUB_TOKEN}@theupdateframework/notary/migrations/server/postgresql\#v0.6.1
+NOTARY_SIGNER_MIGRATION_SOURCE ?= github://holyhope:$${GITHUB_TOKEN}@theupdateframework/notary/migrations/signer/postgresql\#v0.6.1
```

Comment:
There is your account name in this variable, is it intentional?

Reply:
I fixed it.
Also I added environment variables checks in makefile.